app/controllers/regional_organizations_controller.rb

```diff
@@ -53,7 +53,7 @@ def update
 
   def destroy
     @regional_organization = regional_organization_from_params
-    if @regional_organization.is_pending?
+    if @regional_organization.pending?
```

Comment:
Not very firm in ROs. But this might potentially be a DB column...?

Reply:
no, it's defined
```
def pending?
    start_date.nil?
  end
```